### ISSUE TITLE
How add event 'slideChange' on slider?

### ISSUE THREAD

**Issue by begbeder** [surmon-china/vue-awesome-swiper#251]
How add event 'slideChange' on slider?

my code don't working.
`mounted () {
    // instanse this.swiper = this.$refs.mySlider.swiper
     this.swiper.on('slideChange', function () {
        console.log('slide changed')
      })
}`

or 

`swiperOption: {
        on: {
          slideChange () {
            console.log('slideChange')
          }
        }
}`

**Comment by surmon-china** [comment]
1. `<swiper @slideChange="slideChanged">...`

2. `swiperOption.on...`

3. `mounted...`

https://github.com/surmon-china/vue-awesome-swiper/pull/238
https://github.com/surmon-china/vue-awesome-swiper/blob/master/src/swiper.vue#L48

**Comment by begbeder** [comment]
Thanks, but I use "vue-awesome-swiper": "2.6.7".
 I need support ie10.

**Comment by ferdinandfrank** [comment]
@surmon-china The first option doesn't work for me, at least on chrome. The console says the following:
_Event "slidechange" is emitted in component <Swiper> but the handler is registered for "slideChange". Note that HTML attributes are case-insensitive and you cannot use v-on to listen to camelCase events when using in-DOM templates. You should probably use "slide-change" instead of "slideChange"._

If the event name would be 'slide-change' or 'slidechange' it would work. The same applies for all other camelCase event names.

**Comment by surmon-china** [comment]
@begbeder 

Component v2.6.7 uses Swiper3, so you need to use the Swiper3 API to work properly, like:

`swiperOption. onSlideChangeStart = function() { /* something... */ }`

**Comment by surmon-china** [comment]
@ferdinandfrank You describe the problem, I will test it again, thank you for feedback!

**Comment by surmon-china** [comment]
@ferdinandfrank Yes, you are right. You can not catch events while running a vue directly in your browser. The vue project using webpack is normal, as expected, and I will fix this.

**Comment by surmon-china** [comment]
@ferdinandfrank Update to v3.1.1, it will fix it.

**Comment by ferdinandfrank** [comment]
@surmon-china Thank you! :)

**Comment by chairuosen** [comment]
use this to hack it on 2.6.7
```
this.$refs.yourVueSwiperRef.swiper.on('onSlideChangeEnd', () => {
      //this.onSlideChange();
    });
```
